### DISCUSSION TITLE
Mac Studio 192GB vs AMD Ryzen 9 PRO 7945 with 192GB ddr5 7,800MT/s

### DISCUSSION THREAD

**Discussion by hiqsociety** [ggml-org/llama.cpp#3791] [Q&A]
To all the experts here,
Mac Studio 192GB vs AMD Ryzen 9 PRO 7945 with 192GB ddr5 7,800MT/s

i know Mac prompt (input) speed is as fast as it's decoding (so i heard, will this be improved or can this be improved?)

AMD is "cheaper" but what's the inference speed like?
i was wondering should i get 192gb ddr5 which has bandwidth of 51gb/s vs 800gb/s mac studio and i was wondering do i need more ram vs i need faster cpu or at what point will 51gb/s ddr5 ram be "limited" by the processing speed?

Does anyone has some clue to such questions? do we need faster cpu (what's the limit) or do we need faster bandwidth?

**Comment by spirobel** [comment]
faster bandwidth
I have a 7950x ... was a total waste of money. My 3090 is 10x as fast. 
Just get a 3090 and call it a day. :smile:
